Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -9,13 +9,14 @@ var fileEncoding = require('./fileencoding.js');
 
 export async function substituteAppSettingsVariables(folderPath) {
     var configFiles = tl.glob(folderPath + "/**/*.config");
-    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections"];
+    var variableMap = varUtility.getVariableMap();
+    var tags = ["applicationSettings", "appSettings", "connectionStrings", "configSections", "system.webServer"];
```

Comment:
`system.webserver` tag is available in web.config of `ASP4 and ASPCore package`. Since we eliminate environment variables, I assume that its safe to apply substitution for the above tag.